src/substrate/filters/labeler.ts

```diff
@@ -159,7 +159,7 @@ export const Label: LabelerFilter = (
       const { stash, controller, amount } = data;
       return {
         heading: 'Bonded',
-        label: `You bonded ${balanceFormatter(amount)} from controller ${fmtAddr(controller)} to stash ${fmtAddr(stash)}.`,
+        label: `${balanceFormatter(amount)} was bonded by controller ${fmtAddr(controller)} on stash ${fmtAddr(stash)}.`,
```

Comment:
The reason these are "you" is that they're only supposed to be sent to the single address specified in the CWEvent's `includeAddresses`. But if we want these public, then yeah this change makes sense.

Reply:
Indeed we're using them publicly right now. I think we can figure out a way to inject "Notification about your address:" before the label that will let the label be used in both cases.